octoprint_mrbeam/static/js/design_store.js

```diff
@@ -209,7 +223,7 @@ $(function () {
                 error: function (jqXHR, textStatus, errorThrown) {
                     console.error(
                         "Store bought design saving failed with status " +
-                        jqXHR.status,
+                            jqXHR.status,
```

Comment:
There are some formatting differences. Do we have a standard formatting ruleset to verify which one is the correct one?

Reply:
Yea I believe we agreed on "prettier" at one point but it's not used by everyone. I believe we can approve this for now as we will be deciding on new formatters very soon among the team.